README.md

```diff
@@ -10,24 +27,67 @@ A shiny app to provider a user friendly way to QA data files using centralised c
 
 Code used to create functions and variables called by the global, server, and UI scripts can be found in the 'R' folder.
 
-## Packages
+---
+
+## Requirements
+
+
+### i. Software requirements 
+
+- Installation of R 3.6.2 or higher from software centre
+
+- Installation of RTools40 or higher from software centre
+
+### ii. Programming skills required (for editing or troubleshooting)
+
+- R at an intermediate level, [DfE R training guide](https://dfe-analytical-services.github.io/r-training-course/)
+
+  - Particularly [R Shiny](https://shiny.rstudio.com/)
+  
+---
+
+## How to use
+
+
+### Packages
+
 Package control is handled using renv. You will likely need to run `renv::restore()` if this is your first time using the project.
 
-## Tests
+### Tests
+
 Automated tests have been created using testthat and shinytest. All functions created to screen files have an associated failing file, and unit test using testthat, all sharing the same name. There are also a couple of other unit tests of functions in the app. A helper function is used to easily input the associated file and test against it; utilise this when writing any new checks - create the example file first, followed by the unit test, followed by the function. You may then need to update the UI tests if new tests create differences in the outputted page. This is expected due to the snapshot based nature of shinytest.
 
 UI tests have been created using shinytest that test the app loads, that the reset button really does clear everything, and that files that we'd expect to hit each particular stage of the screening do so. More should be added over time as extra features are added.
 
 GitHub Actions provide CI by running the automated tests and checks for code styling, with badges in this README referring to the latest state of the master branch. The yaml files for these workflows can be found in the .github/workflows folder.
 
-## Project functions
+The function `run_tests_locally()` is created in the Rprofile script and is available in the RStudio console at all times to run both the unit and ui tests.
 
-The function `tidy_code()` is created in the Rprofile script and therefore is always available in the RStudio console to tidy code according to tidyverse styling using the styler package. This function also helps to test the running of the code and for basic syntax errors such as missing commas and brackets.
+In the `.github` folder there are workflow files that run the automated tests for all pushes to master and PR's. The automated tests cover everything that is run locally in the 
```

Comment:
Is this missing the end of the sentence?

Reply:
I don't think so? Though it does look like it in the changes. Check the readme on the main repo page on the dev branch

Reply:
Nope, you're right - the merging of master back in probably caused this